doc/source/learning/overview.rst

```diff
@@ -0,0 +1,92 @@
+Overview
+=====================
+
+CaaSP v3 Install
+-----------------
+
+At this moment we need CaaSP v3 for our deployments. This is our SUSE supported Kubernetes
```

Comment:
I think the requirements are already listed in the deployment guide of socok8s.

If necessary we should add links there.